src/style.css

```diff
@@ -1,81 +1,20 @@
 * {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
   font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
 }
 
-ul {
-  list-style: none;
-}
-
-h1 {
-  margin-left: 20%;
-}
-
 .container {
-  width: 50%;
-  margin: 0 auto;
-}
-
-.boxes {
-  display: flex;
-  justify-content: space-between;
-}
-
-.description {
-  display: flex;
-  justify-content: space-between;
-  margin-bottom: 15px;
-}
-
-.box-left,
-.box-right {
-  padding: 5px;
-  width: 55%;
-  margin-top: 10px;
-}
-
-.box-right {
-  width: 45%;
-  margin-left: 40px;
-}
-
-.box-right h3 {
-  margin-bottom: 15px;
-}
-
-.list {
-  border: 1px solid #0b0b0b;
-}
-
-.list li {
-  padding-left: 5px;
-}
-
-.list li:nth-child(even) {
-  background-color: #ddd;
-}
-
-form {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-}
-
-form input {
-  margin-bottom: 5px;
-  font-size: 15px;
-  color: #0b0b0b;
-  outline: none;
+  width: 100% !important;
```

Comment:
[Optional] Using !important rule is a bad practice and should be avoided because it makes debugging more difficult and code less professional.

Reply:
Thank you @bimbolabuari for the review. I am going to remove the ```!important``` to make my codebase more professional. 👍

Reply:
Hey @bimbolabuari , I have just removed it. 👏 👏